runelite-client/src/main/java/net/runelite/client/ui/overlay/OverlayRenderer.java

```diff
@@ -356,6 +398,11 @@ public void keyPressed(KeyEvent e)
 		{
 			inOverlayDraggingMode = true;
 		}
+
+		if (runeLiteConfig.overlayRightClick() && e.isShiftDown())
```

Comment:
you should probably just check VK_SHIFT, and make it constant at top of the class